.github/workflows/release.yml

```diff
@@ -0,0 +1,109 @@
+name: Create Release
+on:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Unit tests
+        uses: ./.github/actions/unit-tests
+
+  release:
+    needs: test
+    name: Release
+    uses: keptn/gh-automation/.github/workflows/release-integration.yml@v1.3.0
+
+  docker_build:
+    needs: [release]
+    name: Docker Build
+    runs-on: ubuntu-20.04
+    env:
+      VERSION: ${{ needs.release.outputs.RELEASE_TAG }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2.4.0
+
+      - name: Load CI Environment from .ci_env
+        id: load_ci_env
+        uses: c-py/action-dotenv-to-setenv@v3
+        with:
+          env-file: .ci_env
+
+      - name: Docker Build
+        id: docker_build_service
+        uses: keptn/gh-automation/.github/actions/docker-build@v1.3.0
+        with:
+          TAGS: |
+            ${{ env.DOCKER_ORGANIZATION }}/${{ env.IMAGE }}:${{ env.VERSION }}
+            ${{ env.DOCKER_ORGANIZATION }}/${{ env.IMAGE }}:${{ env.VERSION }}.${{ env.DATETIME }}
+          BUILD_ARGS: |
+            version=${{ env.VERSION }}
+          REGISTRY_USER: ${{ secrets.REGISTRY_USER }}
+          REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Docker Build Initcontainer
```

Comment:
Nitpick: I would put the init container build step before the main container build step :)

Reply:
Will do in my next PR